synapse/storage/databases/main/purge_events.py

```diff
@@ -325,20 +325,23 @@ async def purge_room(self, room_id: str) -> List[int]:
         # We then run the same purge a second time without this isolation level to
         # purge any of those rows which were added during the first.
 
+        logger.info("[purge] Starting initial main purge of [1/2]]")
         state_groups_to_delete = await self.db_pool.runInteraction(
             "purge_room",
             self._purge_room_txn,
             room_id=room_id,
             isolation_level=IsolationLevel.READ_COMMITTED,
         )
 
+        logger.info("[purge] Starting secondary main purge of [2/2]]")
```

Comment:
think we've got a stray bracket here
```suggestion
        logger.info("[purge] Starting secondary main purge of [2/2]")
```